examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb

```diff
@@ -1,670 +1,764 @@
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/html": [
-       "<div>\n",
-       "<style scoped>\n",
-       "    .dataframe tbody tr th:only-of-type {\n",
-       "        vertical-align: middle;\n",
-       "    }\n",
-       "\n",
-       "    .dataframe tbody tr th {\n",
-       "        vertical-align: top;\n",
-       "    }\n",
-       "\n",
-       "    .dataframe thead th {\n",
-       "        text-align: right;\n",
-       "    }\n",
-       "</style>\n",
-       "<table border=\"1\" class=\"dataframe\">\n",
-       "  <thead>\n",
-       "    <tr style=\"text-align: right;\">\n",
-       "      <th></th>\n",
-       "      <th></th>\n",
-       "      <th>sex</th>\n",
-       "      <th>race</th>\n",
-       "      <th>age_cat</th>\n",
-       "      <th>priors_count</th>\n",
-       "      <th>c_charge_degree</th>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>sex</th>\n",
-       "      <th>race</th>\n",
-       "      <th></th>\n",
-       "      <th></th>\n",
-       "      <th></th>\n",
-       "      <th></th>\n",
-       "      <th></th>\n",
-       "    </tr>\n",
-       "  </thead>\n",
-       "  <tbody>\n",
-       "    <tr>\n",
-       "      <th rowspan=\"5\" valign=\"top\">Male</th>\n",
-       "      <th>African-American</th>\n",
-       "      <td>Male</td>\n",
-       "      <td>African-American</td>\n",
-       "      <td>Less than 25</td>\n",
-       "      <td>0</td>\n",
-       "      <td>M</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>African-American</th>\n",
-       "      <td>Male</td>\n",
-       "      <td>African-American</td>\n",
-       "      <td>Greater than 45</td>\n",
-       "      <td>1 to 3</td>\n",
-       "      <td>M</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>Caucasian</th>\n",
-       "      <td>Male</td>\n",
-       "      <td>Caucasian</td>\n",
-       "      <td>25 - 45</td>\n",
-       "      <td>More than 3</td>\n",
-       "      <td>F</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>African-American</th>\n",
-       "      <td>Male</td>\n",
-       "      <td>African-American</td>\n",
-       "      <td>Less than 25</td>\n",
-       "      <td>1 to 3</td>\n",
-       "      <td>F</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>African-American</th>\n",
-       "      <td>Male</td>\n",
-       "      <td>African-American</td>\n",
-       "      <td>25 - 45</td>\n",
-       "      <td>1 to 3</td>\n",
-       "      <td>F</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Trusted-AI/AIF360/blob/master/examples/demo_lime.ipynb)"
```

Comment:
same here: change to `examples/sklearn/demo_mdss_classifier_metric_sklearn.ipynb`

Reply:
Done